.github/workflows/test.yaml

```diff
@@ -34,7 +34,8 @@ jobs:
       fail-fast: false
       matrix:
         os: ['ubuntu-latest', 'macos-latest', 'windows-latest']
-        python-version: ['3.7', '3.8', '3.9', '3.10']
+        # python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
```

Comment:
Should I uncomment this line, or do we want to be more selective? Like `['3.7', '3.9', '3.11']`

Reply:
Yep 5 of them is a lot :( !

We thought about this for Panel and I suggested to always include the bounds (of course) and not run the CI on one of the latest versions as they are the ones that are less used. In hindsight maybe what we want is actually test more and versions that are less used, as for the other ones it's likely we'll get user reporting problems.

So, well, I don't know :) But this isn't very important in the end to be honest. And whatever subset we pick, we should have the weekly cron job run on the full matrix.

Pick whatever subset you prefer!

Reply:
I have now added `['3.7', '3.9', '3.11']`. Should I take a look at the cron job in a new PR?

Reply:
Yep let's leave that to a new PR. Btw I think we should run the full matrix when a release is made too.